dataplug-instagram/conf/logback.xml

```diff
@@ -30,16 +30,22 @@
   <logger name="com.hubofallthings.dataplugInstagram.apiInterfaces.*" level="INFO" />
   <logger name="com.hubofallthings.dataplugInstagram.controllers.Api" level="INFO" />
   <logger name="com.hubofallthings.dataplugInstagram.apiInterfaces.InstagramFeedInterface" level="DEBUG" />
+  <logger name="com.hubofallthings.dataplugInstagram.apiInterfaces.InstagramProfileCheck" level="DEBUG" />
```

Comment:
Not comfortable with deploying a sandbox version with DEBUG mode on across the board. Great for debugging locally, but deployments (and by extension dev branch version) have to have minimum INFO level on all classes.